packages/fxa-payments-server/src/App.tsx

```diff
@@ -55,39 +58,48 @@ export const App = ({
   FlowEvents.init(queryParams);
   return (
     <AppContext.Provider value={appContextValue}>
-      <AppErrorBoundary>
-        <StripeProvider apiKey={config.stripe.apiKey}>
-          <ReduxProvider store={store}>
-            <Router>
-              <React.Suspense fallback={<RouteFallback />}>
-                {/* Note: every Route below should also be listed in INDEX_ROUTES in server/lib/server.js */}
-                <Route
-                  path="/"
-                  exact
-                  render={() => <Redirect to="/subscriptions" />}
-                />
-                <Route
-                  path="/subscriptions"
-                  exact
-                  render={props => (
-                    <SettingsLayout>
-                      <Subscriptions {...props} />
-                    </SettingsLayout>
-                  )}
-                />
-                <Route
-                  path="/products/:productId"
-                  render={props => (
-                    <SignInLayout>
-                      <Product {...props} />
-                    </SignInLayout>
-                  )}
-                />
-              </React.Suspense>
-            </Router>
-          </ReduxProvider>
-        </StripeProvider>
-      </AppErrorBoundary>
+      <AppLocalizationProvider
+        userLocales={navigator.languages}
+        bundles={['main']}
+      >
+        <Localized id="document" attrs={{ title: true }}>
+          <DocumentTitle title="fIREFOx aCCOUNTs pAYMENTs">
```

Comment:
Was `fIREFOx aCCOUNTs pAYMENTs` to double check this was working? If I saw this while working I'd question its purpose.

Can we remove the `<title>` piece from `public/index.html`?

Also, are there plans for docs? 😁

Reply:
`fIREFOx aCCOUNTs pAYMENTs` style is what we used on lockwise. It's so that we can easily see if a translation is broken in testing before it goes live.

I left the tile in `public/index.html` so that we have something while the initial page load happens.

I should definitely add docs. Though some things may change as I updated all of the other components, so I should probably add after all those are done

Reply:
Can you expand on the capitalization style to test l10n?  I'd expect that to be done with a pseudolocale and not something that a user might see.

Reply:
From what I remember for the decision in lockwise, we want to always be able to catch anything that wouldn't translate before it gets to a user. Something noticeable like this, ensures in the QA process that it shouldn't make it to the user. The reasoning was basically, if non translated text makes it to the user, its wrong either way. Since most of us will be testing in english it'll be easier for these to slip past us

Reply:
If a translation changes, or there's a syntax error when a localizer checks the file in, it falls back to this string, right?  Shouldn't this string be a failsafe?

Reply:
Reverted to english text for these strings. We can use pseudolocalization to track translations in the future. Filed https://github.com/mozilla/fxa/issues/3756